spec/app/domain/authentication/authn_k8s/inject_client_cert_spec.rb

```diff
@@ -16,6 +16,9 @@
   let(:nil_host_id_prefix) { nil }
   let(:csr) { "CSR" }
 
+  let(:default_cert_file_path) { "/etc/conjur/ssl/client.pem" }
+  let(:expected_cert_file_path) { "/etc/conjur/ssl/client.pem" }
```

Comment:
why is this needed? I would guess that not all tests have this as the expected cert path (you must have some tests that use a non-default entry, no?) - and when they don't, the `default_cert_file_path` can be referred to